lib/cli/glob-expression.js

```diff
@@ -7,7 +7,8 @@ async function expandExpression(args) {
         // glob expression ?
         if (_isGlob(expr)){
             // expand expression
-            return await glob(expr, {});
+            const files = await glob(expr, {});
+            return files;
```

Comment:
I thought this wouldn't be necessary anymore with eslint v9 but it's a weird one.

- This way an IDE might complain because the local variable `files` is actually redundant.
- Using `return await glob(expr, {})` instead, eslint then complains about `Redundant use of `await` on a return value`.
- If you were to follow on eslint's advice (i.e. remove `await`) it would in turn complain because `Async arrow function has no 'await' expression`

=> I guess your form is the most useful

Reply:
This is one of the ways.

We can remove  'no-return-await': 2, rule because this one was deprecated in ESLint v8.46.0. 
https://eslint.org/docs/latest/rules/no-return-await

then we can use `return await glob(expression,{});` and no warning will be issued.

Reply:
I guess that would make sense.